### PR TITLE
[PTF python3 migration] Modify test_advanced_reboot PTF script

### PR DESCRIPTION
**Pull request by wsycqyz**
### Description of PR

Summary:
Migrate test_arpall PTF script to Python3

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
This is part of Python3 migration project.
This PR migrate test_arpall PTF script to Python3.

#### How did you do it?
2to3 and manually code change

#### How did you verify/test it?
Test with KVM testbed, physical testbed 3800 and PR test.

### CODE REVIEW

File: tests/common/fixtures/advanced_reboot.py [hunk 1]
```diff
@@ -756,7 +756,8 @@ def __runPtfRunner(self, rebootOper=None):
             params=params,
             log_file='/tmp/advanced-reboot.ReloadTest.log',
             module_ignore_errors=self.moduleIgnoreErrors,
-            timeout=REBOOT_CASE_TIMEOUT
+            timeout=REBOOT_CASE_TIMEOUT,
+            is_python3=True
```
Comment (vaibhavhd) [inline tests/common/fixtures/advanced_reboot.py:760]: Where is this `is_python3` argument being used?

**Review by vaibhavhd — APPROVED** [review]
LGTM, please ensure that the testing is also done on physical DUT, as the KVM test do not cover all the test conditions.

File: tests/common/fixtures/advanced_reboot.py (hunk 1 above)
Comment (wsycqyz) [reply tests/common/fixtures/advanced_reboot.py:760]: In ptf_runner.py, if is_python3==true, Python3 is used to run the PTF script.

**Comment by wsycqyz** [comment]
> LGTM, please ensure that the testing is also done on physical DUT, as the KVM test do not cover all the test conditions.

Understood.

**Review by wangxin — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Review by vaibhavhd — APPROVED** [review]

File: ansible/roles/test/files/ptftests/py3/advanced-reboot.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ def __init__(self):
         self.check_param('lo_v6_prefix', 'fc00:1::/64', required=False)
         self.check_param('arista_vms', [], required=True)
         self.check_param('min_bgp_gr_timeout', 15, required=False)
-        self.check_param('warm_up_timeout_secs', 300, required=False)
+        self.check_param('warm_up_timeout_secs', 600, required=False)
```
[the remark this answers is not in the document]
Comment (wsycqyz) [reply ansible/roles/test/files/ptftests/py3/advanced-reboot.py]: This is part of old commit. Final version didn't have this change.

**Pull request merged**